pq-crypto/bike/utilities.h

```diff
@@ -113,11 +113,11 @@ _INLINE_ uint32_t secure_cmp32(IN const uint32_t v1, IN const uint32_t v2)
 // Return 0 if v1 < v2, (-1) otherwise
 _INLINE_ uint32_t secure_l32_mask(IN const uint32_t v1, IN const uint32_t v2)
 {
-    
+
```

Comment:
Please remove the empty lines (116, 132 and 136).

Reply:
Done